### ISSUE TITLE
Current popup windows

### ISSUE THREAD

**Issue by lovebug0706** [lightbeam24/CustomTube#101]
The share popups or the report button are still the current ones, even on the old themes. Heres 2 images of it in action:
![Screenshot 2023-10-17 110200](https://github.com/lightbeam24/CustomTube/assets/34926216/65e09c84-c905-44f5-9625
![Screenshot 2023-10-17 110152](https://github.com/lightbeam24/CustomTube/assets/34926216/a9df8352-85d2-4b12-987f-84b742ca48ce)
-bdd2ba1f0fff)
Maybe fix these to have the old buttons / menus? Thanks!
![Screenshot 2023-10-17 110852](https://github.com/lightbeam24/CustomTube/assets/34926216/73dcf37a-24c4-44e7-8451-0ced24665105)

**Comment by lightbeam24** [comment]
Yeah, progress hasn't started on these yet. I pretty much never use these features, so they're very low priority unfortunately. Thanks for the screenshots though.
